src/Blazor.Extensions.Notifications.JS/src/NotificationService.ts

```diff
@@ -19,8 +19,7 @@ export class NotificationsManager implements INotificationsManager {
     return false;
   }
 
-  public Create = (title: string, options: object): object => {
-    var note = new Notification(title, options);
-    return note;
+  public Create = (title: string, options: object): void => {
+    new Notification(title, options);
```

Comment:
`System.Text.Json` throws an exception serializing this object which is visible in the browser console. So we don't return it.